blog-pipeline/tests/test_draft_postprocessing.py

```
@@ -0,0 +1,302 @@
+import re
+import sys
+from pathlib import Path
+
+# ---------------------------------------------------------------------------
+# Bootstrap: add blog-pipeline/src to sys.path so imports resolve
+# ---------------------------------------------------------------------------
+_BLOG_SRC = Path(__file__).resolve().parent.parent / "src"
+if str(_BLOG_SRC) not in sys.path:
+    sys.path.insert(0, str(_BLOG_SRC))
+
+from blog_runner._runner import BlogRunner
```

Ruff (0.15.7)

[error] Module level import not at top of file

(E402)